src/RemoteInboxNotifications/SpecRunner.php

```diff
@@ -7,8 +7,8 @@
 
 defined( 'ABSPATH' ) || exit;
 
-use \Automattic\WooCommerce\Admin\Notes\WC_Admin_Note;
-use \Automattic\WooCommerce\Admin\Notes\WC_Admin_Notes;
+use \Automattic\WooCommerce\Admin\Notes\Note;
+use \Automattic\WooCommerce\Admin\Notes\Notes;
```

Comment:
Super nitpick: I wonder if we could add an alias here too.

Reply:
I've only used an alias to `NotesRepository` where there was a name conflict. I would like to rename `Notes` (and other repository type classes) to `NoteRepository` but that was out of scope for this PR (and conflicts with the existing conventions).